### DISCUSSION TITLE
Description in Microsoft Store - USB-DP converter

### DISCUSSION THREAD

**Discussion by ireun** [emoacht/Monitorian#165] [General]
Hello!

I just want to tell You, that "In contrast, an USB to HDMI/DisplayPort converter which supports DDC/CI is not yet known to exist." is not true, I'm using "HP USB-C to DP Adapter P/N 831119-001 ( KSC0044-D25 )" which is working with DDC/CI, and even DaisyChaining. ;)

**Comment by emoacht** [comment]
It's a good news! I will change it in the future update.
I am moving this topic to Discussions for reference to other users.

**Reply by emoacht** [reply]
To be accurate, I think it is "DisplayPort over USB Type-C" a.k.a. "DisplayPort Alt Mode" which uses USB-C connector as DisplayPort. In this mode, USB-C connector and cable is physically used to transmit signals of DisplayPort. It is totally different from the adaptor of older generations.

**Reply by ireun** [reply]
All right, you're most probably right. Though it's making a DP from a USB-C, so kind of converted, not from technological point. :D
